apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -35,9 +205,217 @@ export default class ProjectCard extends React.Component {
       isPublicGallery && isDetailView && projectData.publishedAt;
     const noTimeOnCardStyle = shouldShowPublicDetails ? {} : styles.noTime;
 
+    const {
+      showReportAbuse,
+      isAnyCheckboxSelected,
+      checkboxes,
+      showSubmittedHeader,
+      submitButtonEnabled,
+      reportButtonHovered,
+      reportButtonPressed,
+      showSubmitConfirmation,
+    } = this.state;
+
+    const captchaSiteKey = getScriptData('projects').recaptchaSiteKey;
+
+    const script = document.createElement('script');
+    script.src = 'https://www.google.com/recaptcha/api.js';
+    script.id = 'captcha';
+    script.async = true;
+    script.defer = true;
+    window.onCaptchaSubmit = token => this.onCaptchaVerification(token);
```

Comment:
Does this need to be set on the global `window` object?

Reply:
Hmm it gave me an Uncaught ReferenceError without it

Reply:
Ok yep I think I see it now, so that the captcha knows what to do once you've verified you're a human (we do the same thing in `ReCaptchaDialog`, which is probably where you got it :). Never mind!